libs/common/src/helpers/fee-record-corrections/map-correction-reasons-to-values.test.ts

```diff
@@ -0,0 +1,107 @@
+import { mapCorrectionReasonsToValues, correctionReasonsToValues } from './map-correction-reasons-to-values';
+import { FeeRecordEntityMockBuilder } from '../../test-helpers';
+import { RECORD_CORRECTION_REASON } from '../../constants';
+
+describe('map-correction-reasons-to-values', () => {
+  const feeRecord = new FeeRecordEntityMockBuilder().build();
+
+  describe('correctionReasonsToValues', () => {
+    describe(RECORD_CORRECTION_REASON.FACILITY_ID_INCORRECT, () => {
+      it('should return the value of the fee record property that corresponds to the reason', () => {
```

Comment:
For all these tests please can we name the field that should be returned in the test rather than relying on future maintainers knowing what the 'property that corresponds to the reason' is. e.g.
```suggestion
      it('should return the facility id of the fee record', () => {
```

Reply:
done